src/pyhf/infer/test_statistics.py

```diff
@@ -47,16 +89,88 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
         raise UnspecifiedPOI(
             'No POI is defined. A POI is required for profile likelihood based test statistics.'
         )
+    if par_bounds[pdf.config.poi_index][0] == 0:
+        log.warning(
+            'qmu test statistic used for fit configuration with POI bounded at zero. Use qmutilde.'
+        )
+    return _qmu_like(mu, data, pdf, init_pars, par_bounds)
 
-    tensorlib, optimizer = get_backend()
-    mubhathat, fixed_poi_fit_lhood_val = fixed_poi_fit(
-        mu, data, pdf, init_pars, par_bounds, return_fitted_val=True
-    )
-    muhatbhat, unconstrained_fit_lhood_val = fit(
-        data, pdf, init_pars, par_bounds, return_fitted_val=True
-    )
-    qmu = fixed_poi_fit_lhood_val - unconstrained_fit_lhood_val
-    qmu = tensorlib.where(
-        muhatbhat[pdf.config.poi_index] > mu, tensorlib.astensor(0.0), qmu
-    )
-    return tensorlib.clip(qmu, 0, max_value=None)
+
+def qmu_tilde(mu, data, pdf, init_pars, par_bounds):
+    r"""
+    The test statistic, :math:`\tilde{q}_{\mu}`, for establishing an upper
+    limit on the strength parameter, :math:`\mu` for models with
+    bounded POI.
+
```

Comment:
```suggestion

    Example:
        >>> import pyhf
        >>> pyhf.set_backend("numpy")
        >>> model = pyhf.simplemodels.hepdata_like(
        ...     signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0]
        ... )
        >>> observations = [51, 48]
        >>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
        >>> test_mu = 1.0
        >>> init_pars = model.config.suggested_init()
        >>> par_bounds = model.config.suggested_bounds()
        >>> pyhf.infer.test_statistics.qmu_tilde(test_mu, data, model, init_pars, par_bounds)
        array(3.93824492)

```

Can use the existing `qmu` docstring example here but replace with `qmu_tilde`